tachyon/zk/plonk/examples/BUILD.bazel

```diff
@@ -24,16 +24,46 @@ tachyon_cc_library(
 # TODO(dongchangYoo): This is failed in CI because of timeout, 60 secs.
 # Change |tachyon_cc_test| to |tachyon_cc_unittest| once CI can run within the timeout.
 tachyon_cc_test(
-    name = "examples_unittests",
-    srcs = [
-        "simple_circuit_unittest.cc",
-        "simple_lookup_circuit_unittest.cc",
-        "simple_lookup_v1_circuit_unittest.cc",
-        "simple_v1_circuit_unittest.cc",
+    name = "simple_circuit_test",
+    srcs = ["simple_circuit_unittest.cc"],
```

Comment:
Please rename `simple_circuit_unittest.cc` to `simple_circuit_test.cc` ! It looks like integration test.